stable/redis-ha/values.yaml

```diff
@@ -0,0 +1,79 @@
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  master:
+    requests:
+      memory: 200Mi
+      cpu: 100m
+    limits:
+      memory: 700Mi
+  slave:
+    requests:
+      memory: 200Mi
+      cpu: 100m
+    limits:
+      memory: 200Mi
+  sentinel:
+    requests:
+      memory: 200Mi
+      cpu: 100m
+    limits:
+      memory: 200Mi
+
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+##
+persistentVolume:
+  ## If true, redis will create/use a Persistent Volume Claim
+  ## If false, use emptyDir
+  ##
+  enabled: false
+
+  ## Redis data Persistent Volume access modes
+  ## Must match those of existing PV or dynamic provisioner
+  ## Ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  accessModes:
+    - ReadWriteOnce
+
+  ## Redis data Persistent Volume Claim annotations
+  ##
+  annotations: {}
+
+  ## Redis data Persistent Volume existing claim name
+  ## Requires alertmanager.persistentVolume.enabled: true
+  ## If defined, PVC must be created manually before volume will be bound
+  existingClaim: ""
+
+  ## Redis data Persistent Volume mount root path
+  ##
+  mountPath: /data
+
+  ## alertmanager data Persistent Volume size
+  ##
+  size: 2Gi
+
+  ## Redis data Persistent Volume Storage Class
+  ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
+  ## Default: volume.alpha.kubernetes.io/storage-class: default
+  ##
+  storageClass: ""
+
+  ## Subdirectory of redis data Persistent Volume to mount
+  ## Useful if the volume's root directory is not empty
+  ##
+  subPath: ""
+
+## Redis image version
+redis_image: gcr.io/google_containers/redis:v1
```

Comment:
@joe1chen I pulled the image and did an `docker inspect`:
```
        "Created": "2015-01-22T23:21:05.435496722Z",
        .....
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "REDIS_VERSION=2.8.19",
                "REDIS_DOWNLOAD_URL=http://download.redis.io/releases/redis-2.8.19.tar.gz",
                "REDIS_DOWNLOAD_SHA1=3e362f4770ac2fdbdce58a5aa951c1967e0facc8"
            ],
            "Cmd": [
                "/run.sh"
            ],
            "Image": "49236ba068bbf4458279f55cbad5905bb3d7097278d29d1fce0b956e2d339ee7",
            "Volumes": {
                "/data": {}
            },
            "WorkingDir": "/data",
            "Entrypoint": [
                "sh",
                "-c"
            ],
```
Above is just an excerpt from the inspect with aspects why I think this image is from 2015. Then I looked into the image via `docker run` and found, that the files are

1. from 2015
2. different from the current example (different distribution and different structure)

```
# ls -l /run.sh
-rwxr-xr-x 1 root root 2473 Jan 22  2015 /run.sh
```

:) That surprised me a little; didn't expect that an unmaintained image is located in grc.io/google_containers/. Even the `redis:latest` is deprecated - from 2015-03-20 :(